spec/dummy_app/app/models/custom_id_key_record.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "securerandom"
+
+module PaperTrailCustomId
+  module ModelConfig
+    def primary_key_for_has_many_versions
```

Comment:
My only concern with this test is that it's actually overriding the code for all of the following tests, but I couldn't find a better way to do it. The only other option I can think of would be mocking the `primary_key_for_has_many_versions ` in the `actual test file`. Thoughts?

Reply:
Re-opening `ModelConfig` is definitely a concern for me too. I'll think about it.

Reply:
I have been thinking about this for a while. I think this test just doesn't belong to this gem, instead it belongs to the `paper_trail-configurable_item_id` gem. I feel that's the reason why it looks so bad here. Are there any other tests we can have instead?

Reply:
> I think this test just doesn't belong to this gem, instead it belongs to the `paper_trail-configurable_item_id` gem.

Makes sense to me. 

> Are there any other tests we can have instead?

How about if we delete this file (`models/custom_id_key_record.rb `), and stub `#primary_key_for_has_many_versions`?

```ruby
allow(SomeModel).to receive(:paper_trail).and_return(
  instance_double(
    RecordTrail, 
    primary_key_for_has_many_versions: 'uuid'
  )
)
```

Then, can we test `RecordTrail#record_id` and `Reifier#find_item`?